lib/fishjam_web/controllers/healthcheck_controller.ex

```diff
@@ -23,14 +23,23 @@ defmodule FishjamWeb.HealthcheckController do
 
     conn
     |> put_resp_content_type("application/json")
-    |> render("show.json", report: report)
+    |> send_resp(200, Jason.encode!(%{data: report}))
   end
 
   defp get_health_report() do
     %{
-      status: :up,
+      localStatus: get_health_status(),
+      distributionEnabled: Application.fetch_env!(:fishjam, :dist_config)[:enabled],
```

Comment:
Shouldn't we add here `Map.fetch!` to prevent a situation in which this returns nil?

Reply:
Well there is no need for that based on what is in config reader:
`lib/fishjam/config_reader.ex:208`